src/megaapi_impl.cpp

```diff
@@ -21132,8 +21140,50 @@ void MegaApiImpl::sendPendingRequests()
                 e = API_EARGS;
                 break;
             }
+
+            long long q = request->getTotalBytes();
+            int bs = request->getAccess();
+            int us = request->getNumDetails();
 #ifdef DEBUG
-            client->senddevcommand(command, email);
+            if (strcmp(command, "aodq")
```

Comment:
Maybe, instead of comparing the strings twice, you could define `bool isOdqSubcmd = strcmp(command, "aodq");` and use it here and below :)